src/calva-fmt/src/config.ts

```diff
@@ -11,12 +12,12 @@ const defaultCljfmtContent =
  :align-associative? false}';
 
 const LSP_CONFIG_KEY = 'CLOJURE-LSP';
-let lspFormatConfig: string;
+let lspFormatConfig: string | undefined;
 
 function configuration(workspaceConfig: vscode.WorkspaceConfiguration, cljfmt: string) {
   return {
-    'format-as-you-type': workspaceConfig.get<boolean>('formatAsYouType'),
-    'keep-comment-forms-trail-paren-on-own-line?': workspaceConfig.get<boolean>(
+    'format-as-you-type': !!workspaceConfig.get<boolean>('formatAsYouType'),
+    'keep-comment-forms-trail-paren-on-own-line?': !!workspaceConfig.get<boolean>(
```

Comment:
Using `!!` to ensure we return booleans instead of undefined.

Reply:
I thought I had removed the `format-as-you-type` setting, relying on VS Code's setting for this... But that was probably in the old failed Parinfer branches... Note to self to remove this setting at some point soon.